webapp/helpers.py

```diff
@@ -1,9 +1,19 @@
 import json
 import flask
+import os
+
 from ruamel.yaml import YAML
 from werkzeug.routing import BaseConverter
 
-yaml = YAML(typ="safe")
+
+_yaml = YAML(typ="rt")
+_yaml_safe = YAML(typ="safe")
+
+
+def get_yaml_loader(typ="safe"):
```

Comment:
is there a reason this is `typ` and not `type`? I even added the e to `typ` in this comment before I proof-read

Reply:
It's same as in YAML internally. Weird, but probably best to keep it consistent with what it's used for.

Reply:
`type` is a reserved keyword in python, probably the same reason the yaml lib uses `typ`.